src/foam/u2/Element.js

```diff
@@ -2370,7 +2370,7 @@ foam.CLASS({
 foam.CLASS({
   package: 'foam.u2',
   name: 'CurrencyViewRefinement',
-  refines: 'foam.core.Currency',
+  refines: 'foam.core.UnitValue',
```

Comment:
line 2372 should it be 'UnitValueViewRefinement'?

Reply:
Thanks! changed.